client/src/main/java/org/apache/falcon/client/FalconClient.java

```diff
@@ -1029,6 +1030,12 @@ public String getExtensionDetail(final String extensionName) {
         return getResponse(String.class, getExtensionDetailResponse(extensionName));
     }
 
+    public String getExtensionJobDetails(final String jobName) {
```

Comment:
Shouldn't this be part of AbstractClient and have FalconUnitClient also implement it?